resources/packs/okta/sdk/policies.go

```diff
@@ -0,0 +1,158 @@
+package sdk
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/okta/okta-sdk-golang/v2/okta"
+	"github.com/okta/okta-sdk-golang/v2/okta/query"
+)
+
+// SkdExtension handles cases where Okta's SDK doesn't support a particular API
+type SkdExtension struct {
+	RequestExecutor *okta.RequestExecutor
+}
+
+// Policy wrapper over okta.Policy until okta sdk is complete
+type PolicyWrapper struct {
+	// reuse existing struct from okta sdk
+	okta.Policy
+
+	Settings *PolicySettings `json:"settings,omitempty"`
+}
+
+// MarshalJSON has to handle the custom embedded struct that is based on okta.Policy
+// ideas from https://jhall.io/posts/go-json-tricks-embedded-marshaler/
+func (a *PolicyWrapper) MarshalJSON() ([]byte, error) {
+	policyJSON, err := a.Policy.MarshalJSON()
+	if err != nil {
+		return nil, err
+	}
+
+	var settingsJSON []byte
+	if a.Settings != nil {
+		type Alias PolicySettings
+		type local struct {
+			*Alias
+		}
+		result := local{Alias: (*Alias)(a.Settings)}
```

Comment:
i struggle to understand this, looks like it's just a wrapper around an aliased policy settings.
wouldn't `result := a.Settings` output the same then?